lib/services/services.c

```diff
@@ -241,22 +241,31 @@ resources_action_create(const char *name, const char *standard,
         char *dir = NULL;
         char *buf = NULL;
         struct stat st;
-        for (dir = strtok(strdup(OCF_RA_PATH), ","); dir != NULL; dir = strtok(NULL, ",")) {
+
+        if (dirs == NULL) {
+            services_handle_exec_error(op, ENOENT);
```

Comment:
This would be a case of ENOMEM

As a semi-related aside, at some point I want libcrmservice to be able to set ocf-exit-reason so we can get useful info to the user about such failures